packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -102,10 +117,64 @@ class CupertinoContextMenu extends StatefulWidget {
   CupertinoContextMenu({
     super.key,
     required this.actions,
-    required this.child,
-    this.previewBuilder,
+    this.child,
+    this.builder,
+    this.previewBuilder = _defaultPreviewBuilder,
   }) : assert(actions != null && actions.isNotEmpty),
-       assert(child != null);
+       assert(child == null || builder == null,
+        'Cannot provide both a child and a builder'
+       ),
+       assert(child != null || builder != null,
+        'Either a child or a builder is required'
+       );
+
+  /// Exposes the default border radius for matching iOS 16.0 behavior. This
+  /// value was eyeballed from the iOS simulator running iOS 16.0.
+  ///
+  /// {@tool snippet}
+  ///
+  /// Below is example code in order to match the default border radius for an
+  /// iOS 16.0 open preview.
+  ///
+  /// ```dart
+  /// CupertinoContextMenu(
+  ///   actions: <Widget>[
+  ///     CupertinoContextMenuAction(
+  ///       child: const Text('Action one'),
+  ///       onPressed: () {},
+  ///     ),
+  ///   ],
+  ///   previewBuilder: (BuildContext context, Animation<double> animation, Widget child) {
+  ///     return FittedBox(
+  ///       fit: BoxFit.cover,
+  ///       child: ClipRRect(
+  ///         borderRadius: BorderRadius.circular(CupertinoContextMenu.kOpenBorderRadius * animation.value),
+  ///         child: Image.asset('assets/photo.jpg'),
+  ///       ),
+  ///     );
+  ///   },
+  ///   child: FittedBox(
+  ///     fit: BoxFit.cover,
+  ///     child: Image.asset('assets/photo.jpg'),
+  ///   ),
+  /// )
+  /// ```
+  ///
+  /// {@end-tool}
+  static const double kOpenBorderRadius = _previewBorderRadiusRatio;
```

Comment:
Should we expose the BoxShadow or something for the default shadow as well?

Reply:
Sure, we can do that.